submissions/kolajoTomike/html/index.html

```diff
@@ -8,10 +8,9 @@
 <body>
   <nav>
     <ul >
-      <li><a href="#">Home</a></li>
-      <li><a href="#">Services</a></li>
-      <li><a href="#">Order</a></li>
-      <li><a href="#">Contact Us</a></li>
+      <li><a href="/html/index.html">Home</a></li>
+      <li><a href="/htmlForms/signUp.html">Sign Up</a></li>
```

Comment:
creating navigation links will be done for task 8. It is best if this is done in task 8